### PR TITLE
Update getting started page

### PR DESCRIPTION
**Pull request by zakpatterson**
* Add state tax section with implemented states
* Remove "unlike other software" language
* Add links to social media

### CODE REVIEW

File: src/components/GettingStarted.tsx [hunk 1]
```diff
@@ -80,26 +89,38 @@ export default function GettingStarted(): ReactElement {
         <li>Washington</li>
         <li>Nevada</li>
       </ul>
-
       <p>
-        If your types of income and state residency are supported, you should be
-        able to use UsTaxes to paper file your return!
+        <strong>10</strong>/51 states are supported. If your types of income and
```
Comment (thegrims) [inline src/components/GettingStarted.tsx:93]: Why 51 states here?

Comment (zakpatterson) [reply src/components/GettingStarted.tsx:93]: DC has an income tax return.

Comment (thegrims) [reply src/components/GettingStarted.tsx:93]: Gotcha

**Review by thegrims — APPROVED** [review]
LGTM

**Pull request merged**